### ISSUE TITLE
CLI unusable : ReferenceError: Sandbox is not defined

### ISSUE THREAD

**Issue by xurei** [auth0/wt-cli#105]
After an update of wt-cli, I'm not able to use it anymore.

I've got this error :
```javascript
Sandbox.prototype.updateStorage = function (storage, options, cb) {
^

ReferenceError: Sandbox is not defined
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/sandboxjs/lib/webtask.js:275:1)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/sandboxjs/lib/sandbox.js:8:15)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/wt-cli/node_modules/sandboxjs/lib/index.js:3:15)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

**Comment by xurei** [comment]
I found a workaround by doing those two commands : 
```bash
sudo rm -R /usr/local/lib/node_modules/wt-cli/node_modules/sandboxjs
sudo npm install -g sandboxjs@3.3.0
```

**Comment by ggoodman** [comment]
Hello @xurei. We were too quick in releasing an update to sandboxjs. Your issue is resolved in ebb4d5692c50d3c6d3d3d6f95b0442d31034c0be. Please update your `wt-cli` to `4.0.1`.

Sorry for the inconvenience.
